icicle/tests/test_hash_api.cpp

```diff
@@ -135,21 +169,21 @@ TEST_F(HashApiTest, KeccakLarge)
   auto output_main_case_2 = std::make_unique<std::byte[]>(output_size * config.batch);
   auto output_ref = std::make_unique<std::byte[]>(output_size * config.batch);
 
-  ICICLE_CHECK(icicle_set_device(IcicleTestBase::reference_device()));
+  ICICLE_CHECK(icicle_set_device(s_reference_target));
```

Comment:
all this file messed up the merge.

Reply:
I didnt merge it
Leon is probably taking care of that